src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java

```diff
@@ -1,30 +1,102 @@
 package org.dailyinfo.telegrambot.telegrambot.service;
 
+
+import org.dailyinfo.telegrambot.telegrambot.service.horoscope.*;
+import org.dailyinfo.telegrambot.telegrambot.service.horoscope.Horoscope.*;
 import org.springframework.stereotype.Service;
 import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
 import org.telegram.telegrambots.meta.api.objects.Message;
 import org.telegram.telegrambots.meta.api.objects.Update;
 
 @Service
-public class MessageService {
+public class MessageService extends Horoscope{
```

Comment:
extends?? remove extends from this class

Reply:
I fixed it!